ERROR - 2024-06-06 03:09:24 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(10): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(10): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 {main}
CRITICAL - 2024-06-06 03:09:24 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: jair]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\Home.php(10): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `cliente`;')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->jair()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-06 03:09:31 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(10): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(10): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 {main}
CRITICAL - 2024-06-06 03:09:31 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: jair]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\Home.php(10): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `cliente`;')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->jair()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-06 03:21:03 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(10): CodeIgniter\Database\BaseConnection->query('select * from `...')
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(10): CodeIgniter\Database\BaseConnection->query('select * from `...')
#3 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 {main}
CRITICAL - 2024-06-06 03:21:03 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: jair]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\Home.php(10): CodeIgniter\Database\BaseConnection->query('select * from `cliente`;')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->jair()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-06 03:25:48 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(13): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(13): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 {main}
CRITICAL - 2024-06-06 03:25:48 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: jair]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\Home.php(13): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `cliente`;')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->jair()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-06 03:25:50 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(13): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(13): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 {main}
CRITICAL - 2024-06-06 03:25:50 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: jair]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\Home.php(13): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `cliente`;')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->jair()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-06 03:25:52 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(13): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(13): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 {main}
CRITICAL - 2024-06-06 03:25:52 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: jair]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\Home.php(13): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `cliente`;')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->jair()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-06 03:27:14 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(13): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(13): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 {main}
CRITICAL - 2024-06-06 03:27:14 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: jair]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\Home.php(13): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `cliente`;')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->jair()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-06 03:27:53 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(16): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(16): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 {main}
CRITICAL - 2024-06-06 03:27:53 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: jair]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\Home.php(16): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `cliente`;')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->jair()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-06 03:30:51 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'app_deber_3_6' in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', 'root', '', 'app_deber_3_6', 3306, '', 0)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(16): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'app_deber_3_6' in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(16): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 {main}
CRITICAL - 2024-06-06 03:30:51 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'app_deber_3_6'
[Method: GET, Route: jair]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\Home.php(16): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `cliente`;')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->jair()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-06 03:30:52 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'app_deber_3_6' in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', 'root', '', 'app_deber_3_6', 3306, '', 0)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(16): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'app_deber_3_6' in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(16): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 {main}
CRITICAL - 2024-06-06 03:30:52 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'app_deber_3_6'
[Method: GET, Route: jair]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\Home.php(16): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `cliente`;')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->jair()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-06 03:30:53 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'app_deber_3_6' in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', 'root', '', 'app_deber_3_6', 3306, '', 0)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(16): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'app_deber_3_6' in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(16): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 {main}
CRITICAL - 2024-06-06 03:30:53 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'app_deber_3_6'
[Method: GET, Route: jair]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\Home.php(16): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `cliente`;')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->jair()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-06 03:30:54 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'app_deber_3_6' in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', 'root', '', 'app_deber_3_6', 3306, '', 0)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(16): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'app_deber_3_6' in C:\xampp\htdocs\DeberApi6\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\DeberApi6\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\DeberApi6\app\Controllers\Home.php(16): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(933): App\Controllers\Home->jair()
#4 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\xampp\htdocs\DeberApi6\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\DeberApi6\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\DeberApi6\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\xampp\htdocs\DeberApi6\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 {main}
CRITICAL - 2024-06-06 03:30:54 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'app_deber_3_6'
[Method: GET, Route: jair]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\Home.php(16): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `cliente`;')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->jair()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
